Testscripts/Windows/DiskHotAddRemove-Parallel.ps1

```diff
@@ -24,7 +25,7 @@ function Main {
             if ($storageProfile.OsDisk.ManagedDisk) {
                 # Add managed data disks
                 $storageType = 'Premium_LRS'
-                $diskConfig = New-AzDiskConfig -SkuName $storageType -Location $AllVMData.Location -CreateOption Empty -DiskSizeGB $diskSizeinGB
+                $diskConfig = New-AzDiskConfig -SkuName $storageType -Location $AllVMData.Location -CreateOption Empty -DiskSizeInBytes $diskSizeinGB
                 $dataDisk = New-AzDisk -DiskName $diskName -Disk $diskConfig -ResourceGroupName $AllVMData.ResourceGroupName
```

Comment:
DiskSizeGB => DiskSizeInBytes  why change this?

Reply:
The run time shows the error. It seems to be new change in PS.

Reply:
Please share your running commands, thanks!

Reply:
@juhlee-microsoft You can run your code in the pipeline, maybe your local machine has higher azure ps version?

Reply:
I think you are right. I did not see the warning msg in the pipeline. So we revert this change?